### PR TITLE
Changes protected job for a few stealth modes

### PR DESCRIPTION
**Pull request by rmariani**
I removed Security Officer and Detective as protected jobs from vampire,
changeling, traitor and cult. The reasoning behind protecting jobs in
these rounds is their armory access. Neither detectives nor security
officers have armory access. Expanding possibilities for antagonists is a
good thing in SS13 to keep the air of paranoia alive, so
players can't just rule huge swaths of characters out. Similarly, detective and sec officers don't have power over departments like heads do for cult rounds. If it's safe for HoP, CE or QM to be cultists without a fear of imbalance, these roles will be safer yet.

### CODE REVIEW

**Comment by Shadowmech88** [comment]
> Neither detectives nor security officers have armory access.

Not arguing against the PR itself here, but if a Security Officer asks the HoP for armory access they're probably going to get it, especially if Security is short-staffed.

**Comment by rmariani** [comment]
Yeah, people can acquire access in-round without too much trouble. I do it all the time as traitor. This is a reason why people shouldn't just hand it out, lol.

Keep in mind that HoP is not a restricted job in any of these game modes and it hasn't been problematic.

**Comment by Shadowmech88** [comment]
My point was that Security officers are highly likely to get the access. It's not like any half-decent HoP will hand armory access to a scientist or bartender.
Security Officers may not have armory access by default, but it's so easy for them to acquire it that it's not an issue.

**Comment by rmariani** [comment]
It's far easier for the HoP to get access than a sec officer, though.

**Comment by Shadowmech88** [comment]
True, but the HoP's traitor status is another issue entirely.
I was merely commenting on the changes of this PR.

**Comment by rmariani** [comment]
My point is that if it's not a problem for the HoP being in the pool for these antag modes it won't be a problem for sec officer

**Comment by Shadowmech88** [comment]
Doesn't the HoP have the ability to give himself all-access?
If that's the case, shouldn't the fact that the HoP can be every traitor mean that every job should be able to be every traitor since access level isn't a consideration?

**Comment by clusterfack** [comment]
HoP is already protect from such roles though so ???

**Comment by clusterfack** [comment]
Also roles that start out implanted shouldn't also start as cultists, it's completely nonsense since having an implant prevents you from being converted in the first place.

**Comment by CptWad** [comment]
Yeah, implants are supposed to stop people from being cultists in the first place, having sec traitors is fine because theres no team dynamic but cult would have some HUGE implications should a roundstart sec be cult
in fact, he could probably robust all of his team since it would be such a huge fucking suprise
Also if I recall, HoP was restricted from cult and given an implant because he would hand out all access to the cultists and they would have a free victory

On the topic of ling and vampire
its 2 easy
if you want to win as a ling or vamp, transfer into sec. free greentext
having them start out in sec would be bad
Detective on the other hand would be a fun ling/vamp, since its niche enough that he cant brig people but can still cover his tracks.
In general, yeah having this would be fun, but has larger implications
also lings with thermals in meta is a terrifying prospect
In general I would say :-1: 
maybe once vampire gets nerfed and we see it be a bit harder than id be okay with vampire sec. Ling and cult sec is a solid no for me.

**Comment by CptWad** [comment]
Wait I just checked the code
>Security officers and detectives COULD be lings, just rare

>Also removed IAA's from cult

That's with the whole implant argument. If you are roundstart implanted you should not be a revhead or a cultist. its the rurus 

>Detective and Sec officer removed from protected traitor

actually I am ok with this, part of me wants to still yell implants, but hell more traitor detective and sec officer rounds will inspire people to play it

also [Needs changelog]

**Comment by ghost** [comment]
This is the wrong way to make people play security and an absolutely horrible balance decision. Security antags are incredibly overpowered, nobody will bat an eye on "arrests" and they start with damn near everything to take on murderboning.

The proper way to make people play any job regardless of antag choices is to make the job selector roll antag chances BEFORE JOB SELECTION, meaning everyone first gets picked for antag, then they get assigned appropriate job.

tl;dr don't do this shit.

**Comment by Kurfursten** [comment]
:-1: for implanted roundstart cultists

> The reasoning behind protecting jobs in these rounds is their armory access.

Who said that? The comment in the code you changed said it's due to NT screening of high value jobs.

**Comment by FudgePucker** [comment]
> Armory access

You don't need any of the fancy guns in the armory when you get a taser with our combat system.
Just taser toolbox murderbone.

**Comment by icantthinkofanameritenow** [comment]
@FudgePucker 

> taser toolbox

You mean taser windowslam, right? Toolboxes are worthless for robusting now.

**Comment by PJB3005** [comment]
@icantthinkofanameritenow I semi recently buffed them though...

**Comment by rmariani** [comment]
We have traitor HoPs all the time and none of these doomsday scenarios end up happening. 

People _should_ have to be paranoid about traitor sec officers. This game isn't about balance, it's about just about anything possible. The motivating philosophy should be leaving the door open for tense and wacky fun, not DotA-esque balance concerns.

Is the possibility (_possibility_, requiring robust play) of the security section of a station being corrupted by a cult not thrilling?

Forcing cargo and science into action to counter this is an exhilarating experience, and similar things have happened. It was a blast every single time.

**Comment by Kurfursten** [comment]
Loyalty implant blocks cult conversion: not only does it not make sense for implanted roles to be able to start as cultists, but it's outright bad design. To say that balance doesn't exist, though, that's even more extreme. You even tacitly acknowledge that balance has a line somewhere because you didn't remove restrictions from the Warden and HoS.

This is a pretty controversial change to try to push through as your first ever code contribution, besides.

**Comment by CptWad** [comment]
sec being cult at roundstart is bad because they can arrest someone, drag them over a rune and convert half the station easy
ive done it as a cultist joining sec at roundstart, its bad
really bad
im all for wacky shit but cult sec is not fun its free antag win
I cant imagine what would happen once the HoS realizes one of his officers are cult. That shit will escalate to the HoS roundstart testing all his officers, which is terrible
this game isn't dota but balance is a concern. its not fun if the cult wins all the time. We have the same issue currently with vampire where a vampire can easily become invincible and have a bunch of free ways to get out of trouble for no cost. There's a pull for it right now.
I have been two traitor HoP's. Recently too. It is doomsday-esqe as its fucking piss easy. Each time people complained that the head roles and sec should not be antags because they are supposed to be trusted. cutting off that trust by making them into teamwork antags is not good.
yeah, having your first pull affect 4 roundtypes heavily is a bit over the top. Easy to code but hellofa big change

**Comment by rmariani** [comment]
The change should be considered on its own merits, not my identity. 

And my entire point is that being able to -rule out- huge swaths of the station makes the game less tense and less fun. 

Trust? This isn't a game about having to metagame "trust" people just because of the roles they spawned as. That's ridiculous and you know it. That just discourages competent players who enjoy being antags spawning as important roles.

I am willing to consider that cult isn't great for sec officers. But detective still has bigger hurdles since they don't arrest people and it's considered weird when they do, particularly by other officers. Cult is exponential, and once you can hit a certain threshold, you can be unstoppable, so those concerns make sense.

For the other three roles, there's no in game reasons for sec officer/detective to not be a ling, vampire or traitor. There don't seem to be many convincing balance reasons like there are for cult. Yeah, you can cuff people and drag them away, but I do that as engineer with a stun stick and cable cuffs without trouble. Should engineer be banned too?

**Comment by CptWad** [comment]
Sec and detective CAN be ling or traitor. It'd just rare, what your code does is make it just as likely to happen as other jobs. 
They are "Protected" which means that the game won't hand out ling and traitor to those jobs as much.
"Restricted" means NO ability for those roles to get that antag. IE you can't be a ling MoMMi.

**Comment by rmariani** [comment]
Lowering the chance is just a smaller magnitude of the same stupidity as banning. If sec being a traitor is a catastrophic scenario, then making a smaller chance of such a scenario isn't a sensible solution. But it probably isn't, so it should be evenly distributed.

**Comment by CptWad** [comment]
That's a nice hot opinion but it's there for a reason. If you don't like it, we could try changing the likelihood of it. 
Or just add light protection and put the detective and officer under that. Then they can get their antagship more than others but still have it not upset the balance. Baby steps man. Baby steps.

**Comment by Zth--** [comment]
I'm against this :-1:
I don't even like antag sec

**Comment by rmariani** [comment]
I don't like getting robusted, we shouldn't remove that though. I am going to make a new PR that is the same, except security officers won't be able to be cult. 

Does that sound fair to everyone? Or give me your suggestions otherwise

**Comment by Kurfursten** [comment]
While cult is the main thing I'm concerned with, I think sweeping changes to antag availability should be handled by server-wide vote. Ask one of those abusive admins to set up a poll.

**Comment by CptWad** [comment]
Poll was made in thread. Cult was a majority no.

**Comment by rmariani** [comment]
Even for detective? or just sec officer

**Comment by CptWad** [comment]
implants in general http://strawpoll.me/6535498/r

**Comment by rmariani** [comment]
Ok. I am closing this PR. I will make a new one without the changes to cultist.
